meeting-minutes/2024/2024-08-21.md

```diff
@@ -0,0 +1,129 @@
+| Przemyslaw | Roguski    | Red Hat                                                     | Voting Member             | Yes     |
+| Shridhar   | Chari      | Cisco Systems                                               | Member                    | No      |
+| Sonny      | van Lingen | Huawei Technologies Co., Ltd.                               | Voting Member             | No      |
+| Stefan     | Arntzen    | Huawei Technologies Co., Ltd.                               | Voting Member             | Yes     |
+| Stefan     | Hagen      | Individual                                                  | Secretary                 | No      |
+| Thomas     | Proell     | Siemens                                                     | Member                    | No      |
+| Thomas     | Schaffer   | Cisco Systems                                               | Voting Member             | Yes     |
+| Thomas     | Schmidt    | Federal Office for Information Security (BSI) Germany       | Voting Member             | Yes     |
+| Tobias     | Limmer     | Siemens                                                     | Member                    | No      |
+
+Regrets:
+
+* Stefan Hagen
+
+Quorum was reached (12 voting members present)
+
+## 1.3 Procedures for this meeting (Moderator)
+
+## 1.4 Approval of agenda
+
+## 1.5 Approval of previous minutes (Moderator)
+
+None (motions carried out already per e-mail motions).
+
+## 1.6 Review of action items and resolutions (Secretary Stefan)
+
+n/a
+
+## 1.7 Identification of TC voting members (Secretary)
+
+n/a
+
+### 1.7.1 Prospective voting members attending their first meeting
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+* Feng Cao
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+* Langley Rock
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+### 1.7.5 Members who have declared a leave of absence
+
+* Stegan Hagen
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Secretary)
+
+    ```
+    September 18, 2024
+    October 16, 2024 (daylight saving times still active)
+    November 29, 2024 (normal time switch back done, same time on wall clock)
+    December 18, 2024
+    January 15, 2025
+    ```
+# 3. Discussion
+
+The meeting focused on creating definitions for end of sales and end of life, including the need for a generic definition that covers both hardware and software.
+
+* The team discussed creating a definition for software and hardware end of sales, and decided to use one broader definition that covers both scenarios.
+* They identified the need to define the term "vendor" and discussed integrating the concept of "product family" into the schema.
+* There was a suggestion to use "May" instead of "can" in the standard, and questions were raised about whether cloud services should be covered by open UX.
```

Comment:
```suggestion
* There was a suggestion to use "MAY" instead of "can" in the standard, and questions were raised about whether cloud services should be covered by OpenEoX.